pinot-transport/src/main/java/com/linkedin/pinot/requestHandler/BrokerRequestHandler.java

```diff
@@ -284,19 +294,20 @@ public Object processBrokerRequest(final BrokerRequest request, BucketingSelecti
   private List<String> getMatchedTables(BrokerRequest request) {
     List<String> matchedTables = new ArrayList<String>();
     String tableName = TableNameBuilder.OFFLINE_TABLE_NAME_BUILDER.forTable(request.getQuerySource().getTableName());
-    Map<ServerInstance, SegmentIdSet> routingMap = _routingTable.findServers(new RoutingTableLookupRequest(tableName));
+    Map<ServerInstance, SegmentIdSet> routingMap = _routingTable.findServers(new RoutingTableLookupRequest(tableName,
+        null));
     if (routingMap != null && !routingMap.isEmpty()) {
       matchedTables.add(tableName);
     }
     tableName = TableNameBuilder.REALTIME_TABLE_NAME_BUILDER.forTable(request.getQuerySource().getTableName());
-    routingMap = _routingTable.findServers(new RoutingTableLookupRequest(tableName));
+    routingMap = _routingTable.findServers(new RoutingTableLookupRequest(tableName, null));
```

Comment:
This method actually uses the routing table to get a list of tables that match for a given request. The result would be the same for HLC vs LLC.

I can still pass the routing table parameters (in case we need them for a future option), but it wouldn't change anything in this case.